src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -35,6 +35,10 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
+    <ProjectReference Include="..\Core\Def.Next\ServicesVisualStudio.Next.csproj">
+      <Project>{d8c86336-f959-47f9-9f26-2725b4bc79ff}</Project>
+      <Name>ServicesVisualStudio.Next</Name>
```

Comment:
VisualStudioSetup is a fairly special reference (it's a vsix, without content, etc).

Look instead at out EditorFeatures.Next